licenses/README.rst

```diff
@@ -1,7 +1,9 @@
-********
 Licenses
-********
+========
 
-This directory holds license and credit information for works sunpy is derived from or distributes.
+This directory holds license and credit information for the package,
+works the package is derived from, and/or datasets.
 
-The license file for the sunpy package itself is placed in the root directory of this repository as LICENSE.rst.
+Ensure that you pick a package licence which is in this folder and it matches
+the one mentioned in the top level README.rst file. If you are using the
+pre-rendered version of this template check for the word 'Other' in the README.
```

Comment:
This being in the rendered output is a curious choice on behalf of the template.